src/Plugin/resource/DataProvider/DataProviderFile.php

```diff
@@ -293,4 +294,33 @@ protected function checkUploadErrors($source, array $files) {
     }
   }
 
+  /**
+   * Helper function that checks if a file was uploaded via a POST request.
+   *
+   * @param string $filename
+   *   The name of the file.
+   *
+   * @return bool
+   *   TRUE if the file is uploaded. FALSE otherwise.
+   */
+  protected static function isUploadedFile($filename) {
+    return variable_get('restful_insecure_uploaded_flag', FALSE) || is_uploaded_file($filename);
```

Comment:
actually. This is opening a can of worms :) ppl might start abusing `$_FILES`.

How about adding this hack on an extended class in the restful_test module?